docs/src/whatsnew/latest.rst

```diff
@@ -150,7 +150,10 @@ This document explains the changes made to Iris for this release
 🚀 Performance Enhancements
 ===========================
 
-#. N/A
+#. `@wjbenfold`_ resolved an issue that previously caused regridding with lazy
+   data to take significantly longer than with real data. Benchmark
+   :class:`HorizontalChunkedRegridding.time_regrid_area_w` shows a time decrease
```

Comment:
@wjbenfold We can't actually link to this, but I guess it should be :class:\`benchmarks.HorizontalChunkedRegridding` instead?